sdk/batch/Azure.ResourceManager.Batch/tests/Helpers/ResourceDataHelper.cs

```diff
@@ -69,7 +69,6 @@ public static void AssertAccount(BatchAccountData account1, BatchAccountData acc
             Assert.AreEqual(account1.Tags, account2.Tags);
             Assert.AreEqual(account1.Location, account2.Location);
             Assert.AreEqual(account1.AccountEndpoint, account2.AccountEndpoint);
-            Assert.AreEqual(account1.Identity, account2.Identity);
```

Comment:
if a property is a class instead of primitive type, we should go into the properties of this type.
Like here we could do
```
AssertIdentity(account1.Identity, account2.Identity);
```
where `AssertIdentity` is a new method, like
```
internal void AssertIdentity(ManagedServiceIdentity identity1, ManagedServiceIdentity identity2)
{
// assert all the property of this type here
}
```

Reply:
Understand, here I will choose another parameter to compare